tests/regression/test_pause_resume.py

```diff
@@ -218,18 +214,15 @@ def test_stopped_lido_cant_deposit():
         contracts.lido.deposit(1, 1, "0x", {"from": contracts.deposit_security_module}),
 
 
-@pytest.mark.usefixtures("stopped_lido")
 def test_resumed_staking_can_stake(stranger):
+    contracts.lido.pauseStaking({"from": contracts.agent})
     contracts.lido.resumeStaking({"from": contracts.agent})
     stranger.transfer(contracts.lido, DEPOSIT_AMOUNT)
 
-
 @pytest.mark.usefixtures("stopped_lido")
 def test_resumed_staking_cant_deposit():
```

Comment:
Consider changing name, as it's currently checks that on stopped Lido contract is not possible to resume staking